ctpolicy/ctpolicy.go

```diff
@@ -247,27 +253,24 @@ func (ctp *CTPolicy) getOperatorSCTs(ctx context.Context, cert core.CertDER, exp
 	// Ensure that this channel has a buffer equal to the number of goroutines
 	// we're kicking off, so that they're all guaranteed to be able to write to
 	// it and exit without blocking and leaking.
-	results := make(chan result, len(ctp.operatorGroups))
+	results := make(chan result, len(ctp.logs))
 
 	// Kick off a collection of goroutines to try to submit the precert to each
 	// log operator group. Randomize the order of the groups so that we're not
 	// always trying to submit to the same two operators.
-	for i, group := range ctp.operatorGroups.Permute() {
+	for i, group := range ctp.logs.Permute() {
 		go func(i int, g string) {
 			sctDER, err := getOne(i, g)
-			if canceled.Is(err) {
-				return
-			}
 			results <- result{sct: sctDER, err: err}
 		}(i, group)
 	}
 
-	go ctp.submitPrecertInformational(cert, expiration)
+	go ctp.submitAllBestEffort(cert, true, expiration)
```

Comment:
Rather than declare a pair of constants in this package, I've made it so that the only callers of `submitAllBestEffort` are `SubmitPrecertInformational` and `SubmitFinalCert`, so now it's baked into the name of the function at the callsite. Is that good?

Reply:
Yep, that's good.